_posts/2018-05-25-support-process.markdown

```diff
@@ -0,0 +1,99 @@
+
+## Process Overview
+
+Two engineers are on-call each week. The rotating schedule is published in a google calendar at least a month in advance and engineers are encouraged to trade shifts as necessary.
+
+During work hours, on-call engineers are responsible for responding to issues in our #incidents slack channel. Outside of work hours, they are only responsible for downtime issues.
+
+## On-Call Resonsibilities
+
+While on-call, you are accountable for investigating and fixing timely issues, escalating to additional point-people and/or routing to team-specific backlogs where appropriate.
+
+Your top priority during an on-call period is to address critical issues. Use your judgment about joining ongoing sprint and team activities.
+
+1.  Be available to answer requests and respond to immediate issues
+2.  Investigate and address critical issues using shared playbooks and pulling in point-people where necessary
+3.  Track incidents’ status (we track incidents on a trello board, and if an issue requires attention by a product team we’ll add it to our team’s jira board)
+4.  Improve the support process and resources for the next rotation (this includes our playbooks for fixing issues and docs about the support process)
+
+## Handing Off
+
+We do our handoff after our team-wide standup every Monday. The previous on-call members and the current ones make sure to resolve any outstanding items so the week begins fresh.
+
+# Current State
+
+We're currently in the middle of our first round of on-call shifts and have already had to deal with every edge case/situation we could imagine. The question of "what qualifies as an incident" keeps coming up and we've been steadily refining the process as we go.
+
+As a result of implementing this process, we’ve seen a few wins:
+
+* Since we also archived slack channels that tended to “collect” issues in the past, we’ve successfully aggregated issues into the #incidents channel. Recently, we had a major issue in a shared service that resulted in multiple, disparate effects across our systems. Seeing these all in one place helped us diagnose the underlying problem and provide a consistent message to stakeholders.
+* Engineers get to learn about parts of the Artsy ecosystem that they had been unfamiliar with before. This isn’t trivial and often requires point-people for those systems to help out, but as a result we’ve been able to spread more knowledge and contribute to a shared documentation base.
+* In addition to a single intake for critical issues, we have a single intake for minor requests and bugs that get triaged and addressed by our new product teams. This means it’s more obvious when the same issue comes up, and since this is part of our same ticketing system we use for new products and features, we know the status at all times.
```

Comment:
~~new~~ product teams

(This is the first mention of product teams being "new" and I think it's not necessary.)